src/plugins/viewLargeAction/viewLargeAction.js

```diff
@@ -54,7 +54,7 @@ export default class ViewLargeAction {
 
     return (
       childElement &&
-      !childElement.classList.contains('js-main-container') &&
+      !childElement?.classList.contains('js-main-container') &&
```

Comment:
is there an issue for this?

Reply:
No there isn't an issue. I'm unsure why this issue happened or how to reproduce it too.